### PR TITLE
fix: memory allocation method

### PR DESCRIPTION
**Pull request by i-m-afk**
#### Description of Change
I have fixed the cpp implementation of kruskal.c and changed it such that it can compile on a C compiler

#### Checklist

- [ x] Added description of change
- [ x] Relevant documentation/comments is changed or added
- [x ] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x ] I acknowledge that all my contributions will be made under the project's license.

Notes: 
In C there is no `new` keyword and have to use `malloc` instead.

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Awesome. Thanks! 🚀

File: data_structures/graphs/kruskal.c [hunk 1]
```diff
@@ -27,11 +27,11 @@ struct Graph
 // Creates a graph with V vertices and E edges
 struct Graph *createGraph(int V, int E)
 {
-    struct Graph *graph = new Graph();
+    struct Graph* graph = (struct Graph*)(malloc(sizeof(struct Graph)));
     graph->V = V;
     graph->E = E;
 
-    graph->edge = new Edge[E];
+    graph->edge = (struct Edge*)malloc(sizeof( struct Edge)*E);
```
Comment (StepfenShawn) [inline data_structures/graphs/kruskal.c:34]:
```suggestion
    graph->edge = (struct Edge*)malloc(sizeof(struct Edge) * E);
```

**Review by StepfenShawn — COMMENTED** [review]
Just a few changes about code format. :)

**Review by StepfenShawn — DISMISSED** [review]
thanks!

**Review by tjgurwara99 — DISMISSED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by StepfenShawn — APPROVED** [review]

**Pull request merged**